eclass/qt5-build.eclass

```diff
@@ -58,6 +58,27 @@ esac
 # for tests you should proceed with setting VIRTUALX_REQUIRED=test.
 : ${VIRTUALX_REQUIRED:=manual}
 
+# @ECLASS-VARIABLE: QT5_DOCS
+# @DESCRIPTION:
+# List of docs tarballs that belong to a package. By default set to false.
+: ${QT5_DOCS:=false}
+
+# @ECLASS-VARIABLE: QT5_DOCSMAP
+# @DESCRIPTION:
+# IMPORTANT: Don't forget to update on version bumps!
+# Key:Value pairs of known Qt5 documentation tarballs where key=<version> and
+# value=<upstream timestamp>, sorted by <version>, descending.
+QT5_DOCSMAP=(
+	5.13.1:201907301220
+	5.13.0:201906171600
+	5.12.4:201906140200
+)
+
+# @ECLASS-VARIABLE: QT5_DOCSPV
```

Comment:
uhm, what's this for? I mean, why should ebuilds be concerned? isn't it just another internal variable of the eclass?

Reply:
yes, just internal, just as above didn't really think about that, only having to document it.